enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/OnboardingService.java

```diff
@@ -96,10 +95,6 @@ public void setOnboardingProvider(OnboardingProvider onboardingProvider) {
      */
     @Transactional
     public OnboardingStartResponse startOnboarding(OnboardingStartRequest request) throws OnboardingProcessException, OnboardingOtpDeliveryException, TooManyProcessesException {
-        if (onboardingProvider == null) {
```

Comment:
The main reason for the change is this. We will add another methods, for consent... This PR helps us to avoid null checks.